incubator-tools/date_entities_annotation_tool/date_entities_annotation.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #288.        return json_dict
</pre><p>This whole try-except block is very difficult to read. Please re-organize to avoid so many layers of nesting.</p><p> </p><p>Maybe change this to use if statements in a guard paradigm instead of exceptions</p><p> </p><p><a href="https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3" target="_blank">https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>